From faf58f6627fd0e487986780676b2a798f0e3355a Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 28 Dec 2023 12:16:15 +0100
Subject: [PATCH] com.webos.service.location: Fix paths of config files

These have moved with https://github.com/webosose/com.webos.service.location/commit/a793519c57bc8b2ddfa9bf978266492080b6868b however the references have not been updated accordingly.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 include/GPSPositionProvider.h                            | 2 +-
 include/GPSServiceConfig.h                               | 2 +-
 src/handler/position/network/NetworkPositionProvider.cpp | 2 +-
 src/lunaIpc/LunaLocationServiceUtil.cpp                  | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/GPSPositionProvider.h b/include/GPSPositionProvider.h
index f1c6e87..35459c0 100644
--- a/include/GPSPositionProvider.h
+++ b/include/GPSPositionProvider.h
@@ -53,7 +53,7 @@ using namespace std;
 #define printf_info LS_LOG_INFO
 #define printf_warning LS_LOG_WARNING
 
-#define GPS_CONF_FILE    "/etc/gps.conf"
+#define GPS_CONF_FILE    "/etc/location/gpsConfig.conf"
 
 #define GPS_UPDATE_INTERVAL_MAX     12*60
 
diff --git a/include/GPSServiceConfig.h b/include/GPSServiceConfig.h
index 35ff1fb..061b0bd 100644
--- a/include/GPSServiceConfig.h
+++ b/include/GPSServiceConfig.h
@@ -23,7 +23,7 @@ using namespace std;
 #include <string>
 #include <gps_cfg.h>
 
-#define    GPS_CONF_FILE    "/etc/gps.conf"
+#define    GPS_CONF_FILE    "/etc/location/gpsConfig.conf"
 
 class GPSServiceConfig {
 public:
diff --git a/src/handler/position/network/NetworkPositionProvider.cpp b/src/handler/position/network/NetworkPositionProvider.cpp
index be4b6e8..1c51d12 100644
--- a/src/handler/position/network/NetworkPositionProvider.cpp
+++ b/src/handler/position/network/NetworkPositionProvider.cpp
@@ -26,7 +26,7 @@
 #define URL_LENGTH                             256
 #define WIFI_SERVICE                           "com.webos.service.wifi"
 #define TELEPHONY_SERVICE                      "com.webos.service.telephony"
-#define GEOLOCKEY_CONFIG_PATH                  "/etc/geolocation.conf"
+#define GEOLOCKEY_CONFIG_PATH                  "/etc/location/wsp.conf"
 #define SCAN_METHOD                            "luna://com.webos.service.wifi/scan"
 #define SCAN_PAYLOAD                           "{}"
 
diff --git a/src/lunaIpc/LunaLocationServiceUtil.cpp b/src/lunaIpc/LunaLocationServiceUtil.cpp
index 0f04f04..9a18277 100644
--- a/src/lunaIpc/LunaLocationServiceUtil.cpp
+++ b/src/lunaIpc/LunaLocationServiceUtil.cpp
@@ -19,7 +19,7 @@
 #include "ServiceAgent.h"
 #include <LocationService.h>
 
-#define GEOLOCKEY_CONFIG_PATH       "/etc/geolocation.conf"
+#define GEOLOCKEY_CONFIG_PATH       "/etc/location/wsp.conf"
 
 LocationService *loc_svc_ptr = NULL;
 
